terraform/modules/core/core.tf

```diff
@@ -360,7 +416,7 @@ resource "aws_appautoscaling_policy" "down" {
 
     step_adjustment {
       metric_interval_upper_bound = 0
-      scaling_adjustment          = -1
+      scaling_adjustment          = -2
```

Comment:
I agree in going up by 2 containers at a time.

I wonder if it would be wise to scale down more slowly than we scale up. There's no real cost to us in taking a more cautious approach but there would be a benefit of having resources already in place if the high load is intermittent.

This also relates to the `period` field above on the scale down alarm which is currently 60 seconds: https://github.com/dxw/teacher-vacancy-service/pull/322/files#diff-8a7db2d745b3f970e5c7652791121c2dR343. 

What do you think about scaling down by only 1 container if CPU is normal for a solid 5 minutes?

Reply:
Yep, I'll change that to `-1` 👍